skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/model/Span.java

```diff
@@ -197,4 +201,16 @@ public byte getStatusCode() {
     public String getExceptionStack() {
         return exceptionStack;
     }
+
+    public void addTags(Map<String, String> tags) {
```

Comment:
Why need `addTags`?  `tag` should enough. It causes confuse either using`addTags` or `tag`, and whether have diff-performances.